src/pyodide/internal/snapshot.ts

```diff
@@ -296,9 +296,8 @@ function checkLoadedSoFiles(dsoJSON: DylinkInfo): void {
  * linear memory into MEMORY.
  */
 function makeLinearMemorySnapshot(
```

Comment:
No reason to take importedModulesList as an argument only to return it in the result but then ignore that result object 🤔

Reply:
What a mess.